### PR TITLE
Offset validation

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

**Comment by jlaura** [comment]
Had to do some vis. work to be able to validate the previous PR.  This is the results of that work.  I have a draft iPython notebook for this as well, not included in the PR as it is not ready yet.

File: autocnet/vis/graph_view.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+
+def plot_node(node, ax=None, clean_keys=[], **kwargs):
+    """
+    Plot the array and keypoints for a given node.
+
+    Parameters
+    ----------
+    imageName : str
+                The base name of the image file (without path).
```
Comment (kree) [inline autocnet/vis/graph_view.py]: imageName isn't a parameter, it's pulled out of the node in line 41.

File: autocnet/vis/graph_view.py [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+from matplotlib import pyplot as plt # plotting 
+
+
+def plot_node(node, ax=None, clean_keys=[], **kwargs):
+    """
+    Plot the array and keypoints for a given node.
+
+    Parameters
+    ----------
+    imageName : str
```
Comment (kree) [inline autocnet/vis/graph_view.py]: node should be listed as a parameter.

**Comment by jlaura** [comment]
Updated for comments.

**Pull request merged**